modules/ROOT/pages/mtf-expecting-errors.adoc

```diff
@@ -41,9 +47,11 @@ public void execute(Integer number) {
 }
 ----
 
+You can write tests to validate the expected error type, and error description.
+
 == Expecting an Error Type
```

Comment:
"Expecting" an error means validating that your module actually throws those errors. 
I think it would also be correct to say *Validating an Expected Error Type*.